Defined and request scitt-statement and scitt-receipt media types (#380)
ietf-wg-scitt/draft-ietf-scitt-architecture: draft-ietf-scitt-architecture.md

# IANA Considerations

IANA is requested to register:

> *  the media type application/statement+cose in the "Media Types" registry, see below.
> *  the media type application/receipt+cose in the "Media Types" registry, see below.

Author: ```suggestion
*  the media type application/scitt-statement+cose in the "Media Types" registry, see below.
*  the media type application/scitt-receipt+cose in the "Media Types" registry, see below.
```

# IANA Considerations

> IANA is requested to register:
> 
> *  allocate media type application/signed-statement+cose in the "Media Types" registry, see below.
> *  allocate media type application/receipt+cose in the "Media Types" registry, see below.
> 

Reviewer: ```suggestion
IANA is requested to register:

*  the media type application/signed-statement+cose in the "Media Types" registry, see below.
*  the media type application/receipt+cose in the "Media Types" registry, see below.

```

## COSE Receipts Header Parameter

394 is requested in {{-RECEIPTS}} and has received an early assignment.

## Media Type Registration

> This section register the following media types in the "Media Types" registry:

Reviewer: ```suggestion
IANA is requested to add the following media types to the "Media Types" registry {{!IANA.media-types}}.
```

# IANA Considerations

IANA is requested to register:

*  the media type application/signed-statement+cose in the "Media Types" registry, see below.
*  the media type application/receipt+cose in the "Media Types" registry, see below.

## COSE Receipts Header Parameter

394 is requested in {{-RECEIPTS}} and has received an early assignment.

## Media Type application/statement+cose Registration

IANA is requested to add the following Media-Type to the "Media Types" registry {{!IANA.media-types}}.

> | Name           | Template                   | Reference               |

Author: ```suggestion
| Name           | Template                          | Reference               |
```

> | statement+cose | application/statement+cose | {{signed-statements}} of {{&SELF}} |

Author: ```suggestion
| statement+cose | application/signed-statement+cose | {{signed-statements}} of {{&SELF}} |
```

{: #new-media-types-scitt-statement title="SCITT Signed Statement Media Type Registration"}

{:compact}
Type name:
: application

Subtype name:

> : statement+cose

Author: ```suggestion
: signed-statement+cose
```
